### ISSUE TITLE
Just a question

### ISSUE THREAD

**Issue by poa00** [samfisherirl/AHKv2-with-integration-for-500-popular-Powershell-scripts-Generator#2]
Hi Sam,

Did you code this yourself? It's pretty awesome.

I _just_ became aware of AHK's existence even only a few months ago, when I was (a) generally interested in making a hobby out of coding and (b) looking for a way to make a nice gui snippet manager that I could use for multiple languages, but also as a means to create a nice UI to house vba, that I could then run without having to use a host application.  

Obviously, just a few minutes on AHK's forum was enough to realize that the general consensus is "it's pointless to go through all the trouble, when you can just write macros in ahk and use them in place of the vba." Further, ahk even has a vba to ahk converter - so its doubly pointless.  

Being the stubborn guy that I am, I still wanted to find a way.  Well recently I came across (i) cs-scripts.net by oleg shilo which supports hot-loading *.cs files to run C / VB + and then today I came across (ii) CLR.ahk and to top it off, I run into this repo here as well.  

NOW, seeing this an ahk app that looks like it hosts (at least to some degree) a powershell scripting engine...?!  My pet project is seeming like its more and more feasible.  

TLDR: Is there any documentation, an AHK thread, or even just a post on reddit you can point me to as a good jumping off point to get me acquainted with whatever approach you took to building this?? THANK YOU.
